docs/get-started/alerting.md

```diff
@@ -6,64 +6,161 @@
+
+#### Edit the root notification policy
+
+1. Go to {{icon.bell}} **Alerting > Notification policies** tab.
+2. Click **Edit** on the top right of the root policy box.
+3. Choose whether to keep the default Email contact point, select a new available contact point or create a new one.
+4. In the **Group by** field, specify how alert rules should be processed into notifications. If multiple alerts are matched for this policy, they will be grouped based on the labels you specify, and a notification will be sent per group.
+5. Expand the **Timing options** section and specify how notification wait times should be processed. These are short pauses the system can take to efficiently process multiple sets of alerts for notifications:
+   - **Group wait**: The default is to wait 30 seconds to buffer alerts of the same group before sending a notification initially.
+   - **Group interval**: The default is to wait five minutes before sending a batch of new alerts after the first notification was sent.
+   - **Repeat interval**: The default is to wait four hours before resending an alert.
+6. Click **Save** to save your changes.
+
+#### Create a new notification policy
+
+To create a new notification policy:
+
+1. Go to {{icon.bell}} **Alerting > Notification policies** tab.
+2. Click **New specific policy**.
+3. In the **Matching labels** section define the rules for matching alert labels. A policy will match an alert if the alert’s labels match all the matching labels specified on the policy. If there are no matchers, the policy will handle all the alert instances. For  example, you could add a **node_name=pmm-server** matcher to send out notifications only for this node.
+4. Select an existing contact point for the policy.
+5. Enable **Continue matching subsequent sibling nodes** to continue matching subsequent siblings of the policy after an alert matched the parent policy.
+This can be useful, for example, when you want to send notifications to a catch-all contact point as well as to one of more specific contact points handled by subsequent policies.
+6. Toggle **Override grouping** if you do not want to use root policy grouping.
+7. Toggle **Override general timings** to specify how often you want to wait until the initial notification is sent for a new group. When this is disabled, PMM uses root policy group timings instead.
+8. Add a mute timing policy if you want to mute notifications or this policy for a specific, regular interval. For example, you can create a mute to suppress trivial notifications during weekends.  Mute timings are different from silences in the sense that they are recurring, while silences have a fixed start and end time.
+   
+!!! caution alert alert-warning "Important"
+    Time specified in mute timing must be in UTC and military format i.e. 14:00 not 2:00 PM.
+
 ## Create a Percona templated alert
```

Comment:
(1) I think this should be coupled with "alert rules" since we use the templates to create them line 237
I dont see any guidance here on how to create an alert rule based on PMM templates - why is this here?